### DISCUSSION TITLE
첫 스터디 진행 후 피드백에 대한 공지 사항

### DISCUSSION THREAD

**Discussion by JoisFe** [Study-2-Effective-Java/Effective-Java#33] [2. 공지 사항]
# 첫 스터디 진행 후 피드백에 대한 공지 사항
1. item 담당자 discussion 정리 업로드를 금요일까지 하기
- 해당 item에 대한 질문을 올릴 시간이 부족하게 되기 때문
- 만약 사정으로 인해 업로드 속도가 늦어질 경우 연락 필
2. 스터디 원이 8인으로 늘어남에 따라 item 양을 늘리는 것에 대한 고민
- 일주일 동안 8 item 학습은 힘들다고 판단
- 6 item으로 진행하고 만약 해당 스터디 주 바쁜 사람 제외 혹은 사다리 타기로 6인 정하기
- 전 주 item 담당이 아니였던 인원은 무조건 item 담당하기

**Comment by chikeem90** [comment]
오,, 6 item으로 정했군요 

> 전 주 item 담당이 아니였던 인원은 무조건 item 담당하기

👍
